templates/install/deploy.sh

```diff
@@ -66,28 +66,27 @@ while getopts a:de:gm:n:o:u:t:yvh opt; do
         v)
             set -x
             ;;
-        h)
+        *)
             echo "usage: deploy.sh [options]"
             echo
             echo "deploy the EnMasse suite into a running Kubernetes cluster"
             echo
             echo "optional arguments:"
             echo "  -h                   show this help message"
-            echo "  -a \"none standard\"   Deploy given authentication services (default: \"none\")"
+            echo "  -a TYPE              Deploy given authentication services (default: \"none\")"
+            echo "                         TYPE := { none | standard }"
```

Comment:
It is possible to deploy both, should there by a different syntax for that?

Reply:
Oh I see .. yes. You are right I will change that.